ansible/roles/recovery/files/recovery_dump.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+SECONDS=0
+
+echo "Starting restore of: $1 into $2"
+
+# we have 16 VCPUs on the VMs currently, so hopefuly -j 8 is ok 
+pg_restore -U postgres -d "$2" -j 8 "$1"
```

Comment:
Probably should add `--exit-on-error` and `--clean` (to avoid "already exists" errors)? We should double check which exit codes are returned.

Reply:
But probably that's not a problem if creating the database from scratch